app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -11,7 +11,7 @@
       <td class="ellipsis"><%= link_to task.humanized[:action], defined?(main_app) ? main_app.foreman_tasks_task_path(task.id) : foreman_tasks_task_path(task.id) %></td>
       <td><%= task.state %></td>
       <td><%= task.result %></td>
-      <td><%= _('%s ago') % time_ago_in_words(task.ended_at) %></td>
+      <td><%= task.ended_at ? _('%s ago') % time_ago_in_words(task.ended_at) : _('Paused') %></td>
```

Comment:
I'd probably go with @beav's suggestion and only display a blank field. Displaying `paused` twice in one row looks weird
![screenshot_20170711_153624](https://user-images.githubusercontent.com/7326770/28070895-b8e4f9ac-664e-11e7-9aa1-0296bd92cfe5.png)